source/math/FloatSize3D.ooc

```diff
@@ -79,6 +79,9 @@ FloatSize3D: cover {
 		array := input split(',')
 		This new (array[0] toFloat(), array[1] toFloat(), array[2] toFloat())
 	}
+	linearInterpolation: static func (a, b: This, ratio: Float) -> This {
+		This new(Float linearInterpolation(a width, b width, ratio), Float linearInterpolation(a height, b height, ratio), Float linearInterpolation(a depth, b depth, ratio))
+	}
```

Comment:
`mix` is shorter than `linearInterpolation`, that's for sure. I wouldn't mind `lerp` myself, but it's not up to me, so :)